parm/config/config.defaults.s2sw

```diff
@@ -27,21 +27,22 @@ case "${CASE}" in
     #layout_y_gfs=16
     #WRTTASK_PER_GROUP_GFS=86
     WRTIOBUF="32M"  # TODO: This value is for P8 w/ C384.  Why not update/set this as default in config.fv3 C384?
-    MEDPETS=300  # TODO: P8 wants to use 300 instead of ATMPETS = layout_x * layout_y * 6
+    MEDPETS=300  # TODO: Use 300 instead of ATMPETS = layout_x * layout_y * 6
+    ;;
+  "C768")
+    MEDPETS=300  
     ;;
 esac
 
-# config.ice
-
-# TODO: These also are likely P8 hard-wired configurations.  Perhaps this file should be config.P8.defaults
 # config.wave
-waveGRD='gwes_30m'
-waveinterpGRD=' '
-waveuoutpGRD='gwes_30m'
-MESH_WAV='mesh.gwes_30m.nc'
+
+waveGRD='mx025'
+waveinterpGRD='reg025'
+waveuoutpGRD='mx025'
+MESH_WAV='mesh.mx025.nc'
```

Comment:
Will these work with `C384` on all platforms?
i.e. when configuring an experiment with `--resdet 384` and `--app S2SW`, I would assume this would work.

Reply:
There are not existing ICs but otherwise it should "work" yes.  I can set this as variable for existing ICs to use a different grid or I can create an IC for a test case that uses this.

Reply:
The intent behind the q. is, will this work out of the box on Hera/Orion at the C384 P8 case.
The following is treated as a coupled model forecast-only "test" on any of our 3 platforms (Hera/Orion/WCOSS2)

```python
./setup_expt.py forecast-only --app S2SWA --pslot coupled_test --idate 2013040100 --edate 2013040100 --resdet 384 --comrot /some_large_disk_area/Joe.Schmo/comrot --expdir /some_safe_disk_area/Joe.Schmo/expdir
```
`S2SW`, but same other arguments is used for testing without aerosols.

So, at a minimum, the "newer" `waveGRD` should be added to the cases for `--idate 2013040100`.

Running the HR1 case (C768 coupled + new waves) on WCOSS2 as a test for merging PRs into `develop` is just not feasible.

Reply:
I'll have that IC there for this testing purpose by Monday unless the lower res wave mesh is requested.

Reply:
A low res wave mesh is not required at the moment, because we don't have a case (with the other components) where this will work within the workflow
Perhaps in the near near future.  Thank you.

Reply:
It doesn't make much sense to keep the higher resolution waves when running C384 (which is already too high a resolution for everyday workflow testing, but it is the best we have right now). Just makes testing even slower. Please put the wave grids within a CASE case-block like some of the other settings.

Reply:
I can write here now... I don't know what was wrong before.  mx025 is actually what should be running with waves with C384.  C768 should have higher resolution waves grids (and are a work in progress).   https://github.com/NOAA-EMC/global-workflow/issues/1333 has been created to document the need for lower resolution wave grids for other resolutions.